src/trajectory.jl

```diff
@@ -159,11 +138,86 @@ function transition(rng::AbstractRNG, nt::NoUTurnTrajectory{I}, h::Hamiltonian,
         j = j + 1
     end
 
-    return θ_new, r_new, α / nα
+    H_new = 0 # Warning: NUTS always return H_new = 0;
+    return θ_new, r_new, α / nα, H_new
 end
 
-transition(nt::NoUTurnTrajectory{I}, h::Hamiltonian, θ::AbstractVector{T}, r::AbstractVector{T}) where {I<:AbstractIntegrator,T<:Real} = transition(GLOBAL_RNG, nt, h, θ, r)
+transition(nt::DynamicTrajectory{I},
+        h::Hamiltonian, θ::AbstractVector{T},
+        r::AbstractVector{T}
+    ) where {I<:AbstractIntegrator,T<:Real} = transition(GLOBAL_RNG, nt, h, θ, r)
+
+
+###
+### Find for an initial leap-frog step-size via heuristic search.
+###
+
+function find_good_eps(rng::AbstractRNG,
```

Comment:
I didn't check this as I'm assuming this is just a move of location.

Reply:
nothing was changed here.